src/app/clients/clients.service.ts

```diff
@@ -67,6 +67,23 @@ export class ClientsService {
     return this.http.get(`/clients/${clientId}/charges`, { params: httpParams });
   }
 
+  /**
+   * @param clientId Client Id of payer.
+   * @param chargeId Charge Id of the charge to be paid.
+   */
+  getClientTransactionPay(clientId: string, chargeId: string) {
+    return this.http.get(`/clients/${clientId}/charges/${chargeId}`);
+  }
+
+  /**
+   * @param clientId Client Id of the payment.
+   * @param chargeId Charge Id of the payment.
+   * @param payment  Client Payment data.
+   */
+  payClientCharge(clientId: string, chargeId: string, payment: any) {
+    return this.http.post(`/clients/${clientId}/charges/${chargeId}?command=paycharge`, payment);
```

Comment:
Try to use `httpParams` to specify different fields in the request. This is followed in all the modules and to maintain the consistency in the code.

Ref: In the file, `clients.service.ts` line number 47 of the code in the master branch.